Concepts/Tic-Tac-Toe/Services/GameManager.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TicTacToe.Entities;
+using TicTacToe.Services.Interfaces;
+
+namespace TicTacToe.Services
+{
+    public class GameManager
```

Comment:
Create a print method inside this class and make board an iterator, use this method for printing the grid state.